transformer4planning/preprocess/nuplan_rasterize.py

```diff
@@ -154,42 +154,85 @@ def static_coor_rasterize(sample, data_path, raster_shape=(224, 224),
     aug_current = 0
     aug_rate = kwargs.get('augment_current_pose_rate', 0)
     if "train" in split and aug_rate > 0 and random.random() < aug_rate:
-        augment_current_ratio = kwargs.get('augment_current_pose_ratio', 0.3)
-        augment_current_with_past_linear_changes = kwargs.get('augment_current_with_past_linear_changes', False)
-        augment_current_with_future_linear_changes = kwargs.get('augment_current_with_future_linear_changes', False)
-        speed_per_step = nuplan_utils.euclidean_distance(
-            ego_pose_agent_dic[frame_id // frequency_change_rate, :2],
-            ego_pose_agent_dic[frame_id // frequency_change_rate - 5, :2]) / 5.0
-        aug_x = augment_current_ratio * speed_per_step
-        aug_y = augment_current_ratio * speed_per_step
-        yaw_noise_scale = 0.05  # 360 * 0.05 = 18 degree
-        aug_yaw = (random.random() * 2 - 1) * yaw_noise_scale
-        dx = (random.random() * 2 - 1) * aug_x
-        dy = (random.random() * 2 - 1) * aug_y
-        dyaw = (random.random() * 2 * np.pi - np.pi) * aug_yaw
-        ego_pose_agent_dic[frame_id//frequency_change_rate, 0] += dx
-        ego_pose_agent_dic[frame_id//frequency_change_rate, 1] += dy
-        ego_pose_agent_dic[frame_id//frequency_change_rate, -1] += dyaw
-        aug_current = 1
-        if augment_current_with_future_linear_changes:
```

Comment:
The previous version has two DIFFERENT ways to augment!
New logic only replace the 'augment_current_with_future_linear_changes', not the 'augment_current_with_past_linear_changes'.
Should keep the later one instead of replacing them all.